lib/oban/migration.ex

```diff
@@ -194,6 +194,7 @@ defmodule Oban.Migration do
     case repo().__adapter__() do
       Ecto.Adapters.Postgres -> Oban.Migrations.Postgres
       Ecto.Adapters.SQLite3 -> Oban.Migrations.SQLite
+      _ -> repo().config()[:migrator]
```

Comment:
Having thought about this for a few days, I don't see a cleaner alternative.

* Config doesn't help because Oban isn't running at this point.
* Protocols don't work because adapters are modules, not structs.
* Sticking key on the `config/0` map is much better than some global configuration

We don't want this to return `nil` silently, so at a minimum, this should use `Map.fetch!/2`:

```suggestion
      _ -> Map.fetch!(repo().config(), :migrator)
```